pages/vulnerabilities/Buffer_Overflow.md

```diff
@@ -33,6 +33,10 @@ Vulnerabilities.
 
 A buffer overflow condition exists when a program attempts to put more data in a buffer than it can hold or when a program attempts to put data in a memory area past a buffer. In this case, a buffer is a sequential section of memory allocated to contain anything from a character string to an array of integers. Writing outside the bounds of a block of allocated memory can corrupt data, crash the program, or cause the execution of malicious code.
 
+## NVD Categorization
+
+- [CWE-788: Access of Memory Location After End of Buffer](https://cwe.mitre.org/data/definitions/788.html): This typically occurs when a pointer or its index is incremented to a position after the buffer; or when pointer arithmetic results in a position after the buffer.
```

Comment:
Are these descriptions quoted from the source material? If so I'd suggest they be marked as quotes. Ex:

```suggestion
> [CWE-788: Access of Memory Location After End of Buffer](https://cwe.mitre.org/data/definitions/788.html): This typically occurs when a pointer or its index is incremented to a position after the buffer; or when pointer arithmetic results in a position after the buffer.
```